src/etc/dec2flt_table.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3
```

Comment:
```suggestion
#!/usr/bin/env python
```
Why using version-specific python here? This is not a fixed CI environment.

Reply:
On some platforms, `python` binary/link doesn't exist and it fails to run it.

Reply:
IMO you should create `/usr/local/bin/python` symlink pointing to `/usr/bin/python3` on CI to resemble what modern Linux distributions (ones where `python` is Python 3) do. I'm not an official reviewer though.

Reply:
These scripts (as far as I can tell) aren't run from within CI and are just run manually and rarely when we need to re-generate(?) floating point conversion. 

I think it's fine for them to be python 3 only going forward.